.github/workflows/ci.yml

```diff
@@ -9,19 +9,24 @@ jobs:
   test:
     name: PHP ${{ matrix.php }}
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
 
     strategy:
       fail-fast: false
       matrix:
         php: ['8.0', '8.1']
```

Comment:
@LeoColomb What if instead of this, you just added another item to the list under `include:` for 8.0? Maybe that would get rid of the duplicate 8.1 run.

As for `continue-on-error`, I'm not convinced that that actually prevents reporting the overall run as failed; it just prevents canceling other jobs in the run because that one failed. I think. The docs are unspecific, and reading through https://github.com/actions/toolkit/issues/399 didn't really help either.

Reply:
Good catch, it was just in front of me. Thanks!

Reply:
> I'm not convinced that that actually prevents reporting the overall run as failed

Yes, seems quite a compromised feature 😭 
I'm thinking about just disabling 8.1 for now...